external-libraries.txt

```diff
@@ -174,4 +174,9 @@ Project: Citeproc-Java
 URL:     http://michel-kraemer.github.io/citeproc-java/
 Licence: Apache License, Version 2.0
 
+Id:      com.eclipse.j2v8
+Project: J2V8
+URL:     https://github.com/eclipsesource/J2V8
+Licence: EPL- 1.0
```

Comment:
Could you remove the space before `1.0` so it is a valid [SPDX license identifier]( CC-BY-SA-3.0)?